### PR TITLE
replace localdev orchestrator with docker

### PR DESCRIPTION
**Pull request by butlerx**
This will replace the use of localdev to start each service with docker-compose
localdev will now just be a microservice for loading testdata

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,140 +1,98 @@
...
 
-Next step is to get the Community Platform code cloned and up and running. To do that you clone this repo and use the `localdev` command to both setup and run the code:
+Next step is to get the Community Platform code cloned and up and running. To do that you clone this
+repo and each micro service:
 
 ```
 $ git clone https://github.com/CoderDojo/cp-local-development.git && cd cp-local-development
-$ nvm use
-$ npm install
+$ git clone https://github.com/CoderDojo/cp-zen-platform.git workspace-zen/cp-zen-platform
```
Comment (Wardormeur) [inline README.md]: Covered in "Creating your own forks"

**Comment by Wardormeur** [comment]
Specifying how to switch DB would be nice :)

File: README.md [hunk 2]
```diff
@@ -1,140 +1,98 @@
...
+#### Docker-compose
 
+To first set up the local development environment run
 ```
-Usage "./localdev.js <command>" where command is one of:
-  "init zen": does a fresh setup of your local dev environment
-  "run zen": runs all the services in your system
-  "testdata zen": loads test data for each service
-  "test zen": run the e2e tests
+$ docker-compose run --rm testdata
```
Comment (Wardormeur) [inline README.md]: What's --rm for? 
If that's the test-data db, it should always be wiped before a new test

Comment (butlerx) [inline README.md]: no its the container this just deletes it once its finished running

**Comment by butlerx** [comment]
switch db?

**Comment by Wardormeur** [comment]
test db and "dev" db should be differenciated.
Test db shouldn't be wiped after tests, elsewhat it makes issue replay even more difficult
That's what the --zentest = true flag is for atm, switching db at will.

File: README.md (hunk 1 above)
Comment (butlerx) [inline README.md]: this is about making sure they clone to the write sub dir

File: README.md [hunk 3]
```diff
@@ -1,140 +1,98 @@
...
 ```
 $ git clone https://github.com/CoderDojo/cp-local-development.git && cd cp-local-development
-$ nvm use
-$ npm install
+$ git clone https://github.com/CoderDojo/cp-zen-platform.git workspace-zen/cp-zen-platform
+$ git clone https://github.com/CoderDojo/cp-events-service.git workspace-zen/cp-events-service
+$ git clone https://github.com/CoderDojo/cp-dojos-service.git workspace-zen/cp-dojos-service
+$ git clone https://github.com/CoderDojo/cp-users-service.git workspace-zen/cp-users-service
+$ git clone https://github.com/CoderDojo/cp-badges-service.git workspace-zen/cp-badges-service
+$ git clone https://github.com/CoderDojo/cp-e2e-tests workspace-zen/cp-e2e-tests
```
Comment (butlerx) [inline README.md]: this should be replaced with cp-translation

**Comment by butlerx** [comment]
This is finished now but is depending on [#1148](https://github.com/CoderDojo/cp-zen-platform/pull/1148) and [#72](https://github.com/CoderDojo/cp-badges-service/pull/72) being merged to have a proper test.

**Comment by butlerx** [comment]
id like to remove https://github.com/CoderDojo/cp-local-development/pull/64/files#diff-4e5e90c6228fd48698d074241c2ba760R100 before this pr is merged but it is needed for testing till then

**Pull request merged**